src/warpedview.jl

```diff
@@ -1,62 +1,86 @@
-immutable WarpedView{T,N,A<:AbstractArray,F1<:Transformation,F2<:Transformation,I} <: AbstractArray{T,N}
+_build_extrapolation(etp::AbstractExtrapolation) = etp
+
+function _build_extrapolation{T}(itp::AbstractInterpolation{T}, fill::FillType = _default_fill(T))
+    etp = extrapolate(itp, fill)
+    _build_extrapolation(etp)
+end
+
+function _build_extrapolation{T,N,D<:Union{Linear,Constant}}(parent::AbstractArray{T,N}, degree::D = Linear(), args...)
+    itp = Interpolations.BSplineInterpolation{T,N,typeof(parent),BSpline{D},OnGrid,0}(parent)
+    _build_extrapolation(itp, args...)
+end
+
+immutable WarpedView{T,N,A<:AbstractArray,F1<:Transformation,I,F2<:Transformation,E<:AbstractExtrapolation} <: AbstractArray{T,N}
     parent::A
     transform::F1
-    transform_inv::F2
     indices::I
+    inverse::F2
```

Comment:
That sounds like the sanest approach. We should probably document somewhere that the transformation is effectively applied to the image rather than the coordinates. It's probably more mathematically general/elegant to have it apply to the coordinates, but I agree that intuition goes the other way. And consistency with other languages (assuming Matlab isn't the only one) is a good thing.